src/SvgIcon/SvgIcon.js

```diff
@@ -56,6 +57,10 @@ class SvgIcon extends Component {
     hovered: false,
   };
 
+  shouldComponentUpdate = (nextProps, nextState) => {
```

Comment:
Or we could tell user to use this babel plugin in production and we should be good 😁.